app/src/main/res/layout/list_row.xml

```diff
@@ -3,12 +3,12 @@
     <data>
         <import type="android.view.View" />
         <variable
-            name="singleItemViewModel"
-            type="com.example.marcinwisniewski.intivetrainingapp.movielist.viewmodel.SingleItemViewModel" />
+            name="singleMovieViewModel"
+            type="com.example.marcinwisniewski.intivetrainingapp.movielist.viewmodel.SingleMovieViewModel" />
     </data>
         <LinearLayout
             android:id="@+id/original_linearLayout"
-            android:onClick="@{(v) -> singleItemViewModel.showMovieDetails(singleItemViewModel.movie,v)}"
+            android:onClick="@{(v) -> singleMovieViewModel.showMovieDetails(singleMovieViewModel.movie,v)}"
```

Comment:
Is it necessary to pass singleMovieViewModel.movie as argument?

Reply:
Of course it's not :) i just haven't catch it :)